docs/products/kafka/howto/enable-governance.md

```diff
@@ -68,30 +67,69 @@ Enable governance in Aiven for Apache Kafka® to establish a secure and complian
     updated. Existing topics are not be affected by these changes.
     :::
 
-### Change the default user group
+## Select services for governance
+
+Select the Aiven for Apache Kafka services to include in governance. Topics from these
+services become visible in the Topic Catalog, allowing users to claim ownership of
+existing topics.
+
+1. On the **Administration** page, click <ConsoleLabel name="governance"/>.
+1. In the **Governed services** section, click **Change**.
+1. In the **Select services for governance** dialog, select the services.
+1. Click **Save**.
+
+## Select the governance method
+
+You can manage governance for Aiven for Apache Kafka® service using either the
+Aiven Console or the Terraform Provider.
+
+Depending on your workflow, choose between:
+
+- **Aiven Console**: Manage governance tasks visually through the Aiven Console. View
+  and claim ownership of resources in the Topic Catalog.
+- **Terraform Provider**: Automate governance with the Terraform Provider. It integrates
+  with GitOps workflows and allows governance management across multiple projects.
+
+:::note
+When using Terraform, all governance actions must be performed through Terraform. The
+Aiven Console allows you to view requests but does not support operations such as
```

Comment:
"view requests" might be a bit strong, we have an audit log of the created requests.